packages/engine/Source/Scene/Scene.js

```diff
@@ -4252,6 +4301,116 @@ Scene.prototype.pickVoxel = function (windowPosition, width, height) {
   );
 };
 
+/**
+ * Pick a metadata value at the given window position.
+ *
+ * @param {Cartesian2} windowPosition Window coordinates to perform picking on.
+ * @param {string|undefined} schemaId The ID of the metadata schema to pick values
+ * from, or `undefined` to pick values from any schema.
```

Comment:
The crux here is that the whole picking relies on what the shader-based metadata handling can offer. And right now, accessing a metadata value in the shader is done via 
`vec2 example = fsInput.metadata.exampleVector;`

This does not involve the class name, or the schema ID. (Meaning that the property name is used directly to create that shader line, as of https://github.com/CesiumGS/cesium/pull/12075/files#diff-6709f4b7f058f59687308576a9f30f6657790bf5ebcac5d58a38467c1f0d4f30R531 )

**But**... in order to properly disambiguate metadata properties, the class name **and** schema ID will sooner or later be required - this is tracked in https://github.com/CesiumGS/cesium/issues/10085 

When this is done, it might be a "breaking change". But it's not yet clear whether this will even affect the metadata picking methods. One could still make a case for offering this function with `schemaId===undefined`, meaning "I don't know and I don't care about that schema ID - just give me the metadata value with this class/property name". 

However, I'll try to update the JSDoc to clarify that.